src/app/reducers/index.js

```diff
@@ -1,8 +1,10 @@
 import { combineReducers } from 'redux';
 import FireBaseUserReducer from './firebase_user_reducer';
+import UserCourses from './databse'
```

Comment:
Is there a specific reason as to why you named it "./databse" instead of "./database" (missing an 'a') ? Is the "database" filename reserved or is this there another reason?